specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1003,6 +1915,212 @@ components:
       - "kadaster_natuurlijk_persoon"
       - "ingeschreven_niet_natuurlijk_persoon"
       - "kadaster_niet_natuurlijk_persoon"
+    PrivaatrechtelijkeBeperkingHal:
+      allOf:
+      - $ref: "#/components/schemas/Aantekening"
+      - description: "Een privaatrechtelijke beperking is dat iemand niet alles met een stuk grond of huis mag doen. Of niet alles mag doen met de rechten die hij heeft op de grond of het huis. Deze beperking wordt geregeld volgens het privaatrecht, bijvoorbeeld via een overeenkomst. Soms is er wel een wet nodig die het mogelijk maakt om dit soort beperkingen bij het kadaster in te schrijven. Bijvoorbeeld de Tijdelijke wet huurkoop onroerende zaken."
+        properties:
+          _links:
```

Comment:
identificatie & domein op dit niveau toevoegen. Een privaatrechtelijkebeperking is immers wel los te bevragen.